_generated/MavenAuthenticateV0/package.json

```diff
@@ -21,8 +21,8 @@
     "@types/mocha": "^5.2.7",
     "@types/uuid": "^8.3.0",
     "@types/q": "^1.5.2",
-    "azure-pipelines-tasks-artifacts-common": "^2.212.0",
-    "azure-pipelines-task-lib": "^4.0.0-preview",
+    "azure-pipelines-tasks-artifacts-common": "^2.243.0",
+    "azure-pipelines-task-lib": "4.0.0-preview",
```

Comment:
we removed the ^ which would prevent the updating to future compatible minor/patch. Is this intentional/ do we want to enforce this?

Reply:
As I explained in the description, technically there are some breaking changes from 4.0.0-preview to current latest (4.13?).  It is fixable but don't have the time for it now.

Reply:
I think we should have a work item or something to track this or track the status of the breaking changes in the new tasks